tamr_client/transformations.py

```diff
@@ -0,0 +1,150 @@
+import requests
+from simplejson.errors import JSONDecodeError
```

Comment:
Can you make this a [json.JSONDecodeError](https://docs.python.org/3/library/json.html#json.JSONDecodeError) using the standard library's `json` module rather than `simplejson`? We are trying to remove the dependency on `simplejson` soon.

You should be able to remove the stub you made for `simplejson` if we switch to `json`.